modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/Header.scala

```diff
@@ -32,4 +32,15 @@ object Header {
     _   <- hs.setKeywords(id, bag, false)
   } yield ()
 
+
+  object Defaults {
+    implicit def fromStringOption(v: SeqAction[Option[String]]): SeqAction[String] = v.map(_.getOrElse("No Value"))
```

Comment:
Maybe I didn't understand the question. What do you expect me to do to declare this issue resolved?